src/platform/intel/cavs/platform.c

```diff
@@ -60,11 +60,15 @@ static const struct sof_ipc_fw_ready ready
 		.micro = SOF_MICRO,
 		.minor = SOF_MINOR,
 		.major = SOF_MAJOR,
-#if CONFIG_DEBUG
+#if CONFIG_DEBUG && !DETERMINISTIC_BLD
 		/* only added in debug for reproducability in releases */
-		.build = SOF_BUILD,
+		.build = SOF_BUILD, // build counter
```

Comment:
Until coding style is updated, please use C comments here.

Reply:
I got tired of this so I submitted #3994

Reply:
Ack